pyproject.toml

```diff
@@ -1,12 +1,67 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "setuptools_scm[toml]>=3.4",
-    "scikit-build",
-    "cmake",
-    "ninja; platform_system!='Windows'"
+requires = ["scikit-build-core[pyproject]"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "pylibmagic"
+dynamic = ["version"]
+description = "scikit-build project with CMake for compiling libmagic"
+readme = "README.rst"
+requires-python = ">=3.7"
+authors = [
+    { name = "Giordon Stark", email = "kratsg@gmail.com" },
+]
+classifiers = [
+    "Development Status :: 1 - Planning",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
This was converted from the existing files. But do you not support Python 3.11?

Reply:
probably do, i just never noticed I didn't add it in here

Reply:
```suggestion
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
```

Reply:
need to make sure the badge in readme also is updated for 3.11 too

Reply:
That should happen when there's a release.